README.md

```diff
@@ -88,17 +95,19 @@ finalsetup.yaml
 The work done by each recipe is saved as a `.tar.gz` file. So you would take the last archive, `finalsetup.tar.gz`, to extract on the PineNote. Currently, the archive's size is about 240MB and extracted into the partition would occupy almost 700MB.
 
 ## Install the rootfs on the PineNote
-Basically, you have to extract the `finalsetup.tar.gz` inside the prepared partition. You need to follow Martyn's and Dorian's guides to get to this point.
+Take the `*.tag.gz` file after the latest step and extract it to the prepared partition.
+You need to follow Martyn's and Dorian's guides to get to this point.
```

Comment:
Do you mind adding a link to Dorian's / Martyn's guide?

Reply:
It is in the top of the README already, not sure it's worth repeating: 
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/2690773/207870708-7f8dd703-e4c9-454f-ab96-bd70dc0ceb29.png">